jenkins/ci.opensuse.org/openstack-repocheck.yaml

```diff
@@ -30,6 +30,7 @@
           values:
             - SLE_12_SP2
             - SLE_12_SP3
+            - SLE_12_SP4
```

Comment:
You also need to add something in the `combination-filter` a few lines lower. We will only want to test `Master` on SP4 at first.